webknossos/webknossos/skeleton/group.py

```diff
@@ -63,25 +64,43 @@ def id(self) -> int:
 
     def add_tree(
         self,
-        name: str,
+        name_or_tree: Union[str, Tree],
```

Comment:
strictly speaking this is a breaking change, because code that used `add_tree(name=...)` won't work now, anymore. However, I couldn't find such code in the SCM organization (since it's the first parameter this seems unusual, anyway). So, I'd think, it's fine to do this change here. Do you agree?